blueprints/acceptance-test/files/tests/acceptance/__name__-test.js

```diff
@@ -1,4 +1,4 @@
-/* jshint expr:true */
+<% if (jshintExists) { %>/* jshint expr:true */<% } %>
```

Comment:
I'm not following. Can you show me an example of what you mean? The following will output the unnecessary line break as well.
```
<% if (jshintExists) { %>/* jshint expr:true */
<% } %>
```